package.json

```diff
@@ -121,7 +121,7 @@
     "nedb-promise": "^2.0.1",
     "node-fetch": "^1.7.3",
     "pg": "^7.4.0",
-    "qweb3": "^0.6.0",
+    "qweb3": "^0.7.0",
```

Comment:
0.7.1?

Reply:
was waiting for qweb3 pr to be merged in first. updated